index.html

```diff
@@ -12360,7 +12360,7 @@ <h2>Definitions of States and Properties (all aria-* attributes)</h2>
   &lt;li role="option" aria-setsize="16" <strong>aria-posinset="7"</strong>&gt; cantaloupes &lt;/li&gt;
   &lt;li role="option" aria-setsize="16" <strong>aria-posinset="8"</strong>&gt; dates &lt;/li&gt;
 &lt;/ul&gt;</pre>
-				<p>Authors MUST set the value for <pref>aria-posinset</pref> to an integer greater than or equal to 1, and less than or equal to the size of the set when that size is known. Authors SHOULD use <pref>aria-setsize</pref>.</p>
+				<p>Authors MUST set the value for <pref>aria-posinset</pref> to an integer greater than or equal to 1, and less than or equal to the size of the set when that size is known. If <pref>aria-posinset</pref> is specified, authors MUST specify a value for <pref>aria-setsize</pref>.</p>
```

Comment:
Because it is not always required, the normative requirement needs a conditional phrase at the beginning. Otherwise, it can be read as requiring the attribute. The requirement is really only about the range of allowed values.

Example:

```suggestion
				<p>When specifying <pref>aria-posinset</pref>, Authors MUST specify a value that is an integer greater than or equal to 1, and less than or equal to the size of the set when that size is known. If authors specify <pref>aria-posinset</pref>,  authors MUST also specify a value for <pref>aria-setsize</pref>.</p>
```